packages/lesswrong/components/posts/PostsPage/PostsPageTwo.jsx

```diff
@@ -0,0 +1,306 @@
+      textAlign: 'center',
+    },
+    mobileVote: {
+      position: 'absolute',
+      top: -14,
+      right: 0,
+      fontSize: '2em',
+      display: 'inline-block',
+      textAlign: 'center',
+    },
+    mobileDate: {
+      marginLeft: 20,
+      display: 'inline-block',
+      color: theme.palette.grey[600]
+    },
+    mobileActions: {
+      display: 'inline-block',
+      marginLeft: 15,
+      color: theme.palette.grey[600],
+    },
+    mobileDivider: {
+      width: '60%',
+      marginTop: 20,
+      marginLeft: 0,
+      borderColor: theme.palette.grey[100]
+    },
+    postBody: {
+      marginBottom: 50,
+    },
+    postContent: postBodyStyles(theme),
+    subtitle: {
+      ...theme.typography.subtitle,
+    },
+    voteBottom: {
+      position: 'relative',
+      fontSize: 42,
+      textAlign: 'center',
+      display: 'inline-block',
+      marginLeft: -4,
+      marginRight: 15,
+    },
+    postFooter: {
+      marginBottom: 30,
+    },
+    draft: {
+      color: theme.palette.secondary.light
+    },
+    recommendedReading: {
+      width: 640,
+      margin: 'auto',
+      [theme.breakpoints.down('sm')]: {
+        width:'100%'
+      }
+    },
+    moderationGuidelinesWrapper: {
+      width: 'calc(100% - 61px)',
+      verticalAlign: 'top',
+      display: 'inline-block',
+      ...commentBodyStyles(theme)
+    },
+    inline: {
+      display: 'inline-block'
+    },
+
+})
+
+class PostsPage extends Component {
+
+  render() {
+    const { loading, document, currentUser, location, router, classes, params } = this.props
+    const { PostsPageTitle, PostsAuthors, HeadTags, PostsVote, SmallMapPreviewWrapper,
+      LinkPostMessage, PostsCommentsThread, Loading, Error404, PostsGroupDetails, RecommendedReadingWrapper,
+      PostsTopSequencesNav, PostsPageMetadata, ModerationGuidelinesBox, HorizontalPostsVote, FromNowDate,
+      PostsPageMobileActions } = Components
+
+    if (loading) {
+      return <div><Loading/></div>
+    } else if (!document) {
+      return <Error404/>
+    } else {
+      const post = document
+      let query = location && location.query
+      const view = _.clone(router.location.query).view || Comments.getDefaultView(post, currentUser)
+      const description = post.plaintextExcerpt ? post.plaintextExcerpt : (post.body && post.body.substring(0, 300))
+      const commentTerms = _.isEmpty(query && query.view) ? {view: view, limit: 500} : {...query, limit:500}
+      const sequenceId = params.sequenceId || post.canonicalSequenceId;
+
+      return (
+        <div className={classes.root}>
+          <HeadTags url={Posts.getPageUrl(post, true)} title={post.title} description={description}/>
+
+          {/* Header/Title */}
+          <div className={classes.header}>
+            <PostsTopSequencesNav post={post} sequenceId={sequenceId} />
+            <PostsPageTitle post={post} />
+            <span className={classes.mobileVote}>
+              <Hidden mdUp implementation="css">
```

Comment:
Can these be merged into `<Hidden mdUp implementation="css" className={classes.mobileVote}>`? If so, that would be a modest readability improvement.

Reply:
Alas, they cannot, because the class changes the `display` property, which the Hidden component uses to hide the component, and so you can't override it.